bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -32,6 +32,10 @@ def process
 
           private
 
+          def valid_source_file?(source_file_path)
+            !I18nScriptUtils.unit_directory_change?(I18N_SOURCE_DIR_PATH, source_file_path)
+          end
+
```

Comment:
Here we are saying that a source file is valid if it's not a copied version of a file in a different directory. 
What happens if the valid source file is updated? Since it would already exist in the source directory, would it be skipped?

Reply:
Correct, updating the previous version of the file will be skipped until we resolve this manually